key.core/src/main/java/de/uka/ilkd/key/util/MiscTools.java

```diff
@@ -59,10 +25,27 @@
 import de.uka.ilkd.key.rule.Rule;
 import de.uka.ilkd.key.rule.RuleApp;
 import de.uka.ilkd.key.speclang.LoopSpecification;
+import org.key_project.util.Filenames;
+import org.key_project.util.Strings;
+import org.key_project.util.collection.*;
 import recoder.io.ArchiveDataLocation;
 import recoder.io.DataFileLocation;
 import recoder.io.DataLocation;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.zip.ZipFile;
+
 /**
```

Comment:
Again: Should spotless not take care of these irrelant changes in imports?

Reply:
It does not, there isn't even a warning.

There is a huge amount of files with unused imports (I cleaned most of them up I think).

Reply:
Why does it not look at that? It can't or it is not configured to do?

Reply:
I don't know whether it is configured to do that. [Here](https://github.com/diffplug/spotless/blob/main/plugin-maven/README.md#java) is an example how to configure it, which I think we should to keep the order consistent, whatever order we decide on (e.g. first current project, then external libraries, finally standard library). 
@WolframPfeifer